### DISCUSSION TITLE
[A/L] (2021/22) Relazioni - Esercizio 9

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#281] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152319127-8bf27dce-2797-4fec-92db-c7ef95e8fc9b.png)

**Comment by LuginiAndrea** [comment]
$R$ non è riflessiva: presi $a = 100, b = 10$ abbiamo che $a \geq b$ ma non abbiamo che $a \leq b$
$R$ è transitiva: consideriamo $\left(a,b\right)R\left(c,d\right)$ e $\left(c,d\right)R\left(w,z\right)$:
 $\left(a \geq b \wedge c \leq d\right) \wedge \left(c \geq d \wedge w \leq z\right) \Longrightarrow \left(a \geq b\right) \wedge \left(w \leq z\right) \Longrightarrow \left(a,b\right)R\left(w,z\right)$. Intuitivo in quanto ogni coppia nella relazione non dipende dall'altra. Notare inoltre che se sussite la transitività, implichiamo che $c = d$.
 $R$ non è anti-simmetrica: Prendiamo $\left(10,10\right)R\left(5,5\right)$. Vediamo subito come vale anche $\left(5,5\right)R\left(10,10\right)$ ma $10 \neq 5$
 $R$ non è simmetrica a meno che $a = b \wedge c = d$. Basti pensare che se un $a > b$ allora non può essere che $b > a$
 $R$ è totale: presa una qualunque coppia di numeri, $a \geq b$ o $b \geq a$, quindi qualsiasi coppia si trova per forza o sotto a sopra con qualsiasi altra coppia

**Comment by Elia-Belli** [comment]
Concordo con @LuginiAndrea, tranne che sulla totalità.

**Definizione Totalità**: una relazione $R \subseteq A\times A$ è totale se comunque scelti $a,b \in A$ esiste $aRb \lor bRa$.
"Comunque scelti" significa che deve valere anche per $a=b$, ovvero per essere totale deve valere $aRa$, cioè $R$ deve essere riflessiva.

Abbiamo però dimostrato che $R$ non è riflessiva per controesempio, allora $R$ non è nemmeno totale.

**Comment by LuginiAndrea** [comment]
> Concordo con @LuginiAndrea, tranne che sulla totalità.
> 
> **Definizione Totalità**: una relazione $R \subseteq A\times A$ è totale se comunque scelti $a,b \in A$ esiste $aRb \lor bRa$.
> "Comunque scelti" significa che deve valere anche per $a=b$, ovvero per essere totale deve valere $aRa$, cioè $R$ deve essere riflessiva.
> 
> Abbiamo però dimostrato che $R$ non è riflessiva per controesempio, allora $R$ non è nemmeno totale.
> 

Hai ragione, scusate la mia "defaillance"

**Comment by LuginiAndrea** [comment]
> Concordo con @LuginiAndrea, tranne che sulla totalità.
> 
> **Definizione Totalità**: una relazione $R \subseteq A\times A$ è totale se comunque scelti $a,b \in A$ esiste $aRb \lor bRa$.
> "Comunque scelti" significa che deve valere anche per $a=b$, ovvero per essere totale deve valere $aRa$, cioè $R$ deve essere riflessiva.
> 
> Abbiamo però dimostrato che $R$ non è riflessiva per controesempio, allora $R$ non è nemmeno totale.
> 

Hai ragione, scusate la mia "defaillance"

**Comment by LuginiAndrea** [comment]
> $R$ non è riflessiva: presi $a = 100, b = 10$ abbiamo che $a \geq b$ ma non abbiamo che $a \leq b$
> $R$ è transitiva: consideriamo $\left(a,b\right)R\left(c,d\right)$ e $\left(c,d\right)R\left(w,z\right)$:
>  $\left(a \geq b \wedge c \leq d\right) \wedge \left(c \geq d \wedge w \leq z\right) \Longrightarrow \left(a \geq b\right) \wedge \left(w \leq z\right) \Longrightarrow \left(a,b\right)R\left(w,z\right)$. Intuitivo in quanto ogni coppia nella relazione non dipende dall'altra. Notare inoltre che se sussite la transitività, implichiamo che $c = d$.
>  $R$ non è anti-simmetrica: Prendiamo $\left(10,10\right)R\left(5,5\right)$. Vediamo subito come vale anche $\left(5,5\right)R\left(10,10\right)$ ma $10 \neq 5$
>  $R$ non è simmetrica a meno che $a = b \wedge c = d$. Basti pensare che se un $a > b$ allora non può essere che $b > a$
